tomviz/python/GaussianFilterTiltSeries.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name" : "GaussianFilter",
+  "label" : "Gaussian Filter",
```

Comment:
As described in https://github.com/OpenChemistry/tomviz/blob/master/tomviz/python/Operators.md, `name` is the internal name for the Operator, and `label` is the name that appears in the UI. `name` must not contain spaces.

Editing these fields should not cause the UI to disappear unless the JSON becomes ill formed. In that case, you should get an error message indicating the JSON is invalid.